packages/desktop/src/renderer/components/Sidebar/ChannelsPanel/ChannelsPanel.tsx

```diff
@@ -2,24 +2,33 @@ import React from 'react'
 import Grid from '@mui/material/Grid'
 import List from '@mui/material/List'
 import { useModal } from '../../../containers/hooks'
-import { PublicChannel } from '@quiet/types'
+import { PublicChannel, UserProfile } from '@quiet/types'
 import SidebarHeader from '../../ui/Sidebar/SidebarHeader'
 import ChannelsListItem from './ChannelsListItem'
+import UserProfileListItem from './UserProfileListItem'
 
 export interface ChannelsPanelProps {
   channels: PublicChannel[]
+  myUserProfile?: UserProfile
+  userProfiles: Record<string, UserProfile>
+  connectedPeers: string[]
   unreadChannels: string[]
   setCurrentChannel: (id: string) => void
   currentChannelId: string
   createChannelModal: ReturnType<typeof useModal>
+  isTorInitialized: boolean
 }
 
 const ChannelsPanel: React.FC<ChannelsPanelProps> = ({
   channels,
+  myUserProfile,
+  userProfiles,
+  connectedPeers,
```

Comment:
A note: when we add QSS support we'll need to change the online status display to be based on some kind of heartbeat "i'm online" message (an orbitdb entry that for now can last forever but eventually will need a TTL) because we won't always be connecting to user devices directly.